main.go

```diff
@@ -75,14 +75,17 @@ Docker Image: quay.io/mongodb/mongodb-atlas-service-broker`
 }
 
 func startBrokerServer() {
+	// Administrators can control what providers/plans are available to users
+	providersConfig := getEnvOrDefault("PROVIDERS_CONFIG_FILE", "")
```

Comment:
If the user provides an environment variable that doesn't point at a valid JSON file we should panic.